06.bowling_object/game.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require_relative 'frame'
+
+class Game
+  def initialize(marks)
+    @frames = []
+    mark_idx = 0
+    0.upto(Frame::LAST_FRAME) do |frame_idx|
+      frame = Frame.new(frame_idx)
+      mark_idx += frame.add_shot(marks[mark_idx..])
```

Comment:
```suggestion
      frame = Frame.new(frame_idx, marks[mark_idx..])
      mark_idx += frame.scores.size
```

add_shotを呼び忘れると正しいscoreが算出できないクラスよりも、newする段階でadd_shotしてしまう方が使い手に優しいかも？
https://qiita.com/jnchito/items/49fe82364bca2f4f37f5